Kajabi/sage-lib
# Tabs - update responsive styles

## Description

* - [x] - update responsiveness of sage-tabs

### Screenshots


|  before  |  after  |
|--------|--------|
|![Screen Shot 2020-12-18 at 2 37 19 PM](https://user-images.githubusercontent.com/1241836/102659221-ab246900-413e-11eb-8b3d-45b1216c926b.png)|![Screen Shot 2020-12-18 at 2 38 00 PM](https://user-images.githubusercontent.com/1241836/102659239-b37ca400-413e-11eb-8570-fe45a3e667d4.png)|


## Test notes


### Steps for testing
1. Visit sage tabs page on a viewport < `767px`: http://localhost:4000/pages/object/tabs
2. Verify that the choice tabs have the `.sage-tabs--choice` modifier and that stack to reduce overflow


## Related

- [BUILD-553](https://kajabi.atlassian.net/browse/BUILD-553)
